server/lib/pdf.js

```diff
@@ -0,0 +1,28 @@
+import config from 'config';
+import fetch from 'node-fetch';
+
+import { TOKEN_EXPIRATION_LOGIN } from '../lib/auth';
+
+export const getTransactionPdf = async (transaction, user) => {
+  if (['ci', 'test'].includes(config.env)) {
+    return;
+  }
+  const pdfUrl = `${config.host.pdf}/transactions/${transaction.uuid}/invoice.pdf`;
+  const accessToken = user.jwt({}, TOKEN_EXPIRATION_LOGIN);
+  const headers = {
+    Authorization: `Bearer ${accessToken}`,
+  };
+  return await fetch(pdfUrl, { method: 'get', headers })
+    .then(response => {
+      const { status } = response;
+      if (status >= 200 && status < 300) {
+        return response.body;
+      } else {
+        console.warn('Failed to fetch PDF');
+        return null;
+      }
+    })
+    .catch(error => {
+      console.error(`Error fetching PDF: ${error}`);
```

Comment:
It's better to use `logger` on the API. You can import it from `server/lib/logger`.

```suggestion
      logger.error(`Error fetching PDF: ${error}`);
```